.github/ci/generate-news-feed.js

```diff
@@ -0,0 +1,83 @@
+import { Feed } from 'feed';
+import fs from 'fs';
+
+const endpoint = 'https://www.keysight.com/blogs/query-index.json';
+const feedInfoEndpoint = 'https://www.keysight.com/blogs/feed-info.json';
```

Comment:
what does this do? Is this a spreadsheet we need to add somewhere?

Reply:
We might be able to get this from the feed-info.xlsx.